manual/Creating_a_table/Creating_a_sharded_table/Table_sharding_current_limitations.md

```diff
@@ -0,0 +1,14 @@
+# Manual Table Sharding: Current Limitations
+
+When creating a sharded table, you should be aware of the current limitations you will face while working with it.
+
+1. You are not allowed to mix local table creation on the first server in the cluster and a clustered table on the second cluster. When you create a sharded table, you should try to use the same node you used for creation to interact with and send all queries to it without mixing different nodes. This limitation is due to an issue when we initialize the internal map of the shards once you create a clustered table because of how replication works, it will rewrite all data.
+
+2. You need to have an active and running Buddy to use the auto-sharding feature. It will not work without Buddy running.
```

Comment:
Move this to the beginning of the "Creating a Sharded Table" like we do for other buddy-related features.